ui/apps/platform/cypress/constants/NetworkPage.js

```diff
@@ -20,9 +20,9 @@ export const selectors = {
     simulatorSuccessMessage: 'div[data-testid="message-body"]:contains("Policies processed")',
     panels: networkPanels,
     legend: {
-        deployments: '[data-testid="deployment-legend"] div',
-        namespaces: '[data-testid="namespace-legend"] div',
-        connections: '[data-testid="connection-legend"] div',
+        deployments: '[data-testid="deployment-legend"] li',
+        namespaces: '[data-testid="namespace-legend"] li',
+        connections: '[data-testid="connection-legend"] li',
```

Comment:
nit: We've got a dev-defined precise selector target, but we still have to switch from one brittle child selector to another.

Reply:
Good call, I've updated this to be selector agnostic and just check "the n-th item in the legend". I also update this to use less Cypress chaining, which is more in line with how we have been doing recent tests.